app/controllers/homepage_controller.rb

```diff
@@ -139,6 +140,13 @@ def index
   # rubocop:enable AbcSize
   # rubocop:enable MethodLength
 
+  include ActionView::Helpers::JavaScriptHelper
+
+  def custom_head_scripts
```

Comment:
I guess these changes should be in landing_page_controller?

Reply:
@sktoiva @valdis Is this still a relevant change request?

Reply:
@sktoiva @thomasmalbaux issues resolved